packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -169,9 +187,42 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _motionManager = [[CMMotionManager alloc] init];
   [_motionManager startAccelerometerUpdates];
 
-  if (![self setCaptureSessionPreset:_resolutionPreset withError:error]) {
-    return nil;
+  NSError *outError = nil;
+
+  if (_fps) {
+    // The frame rate can be changed only on a locked for configuration device.
+    if ([_captureDevice lockForConfiguration:&outError]) {
+      [_videoCaptureSession beginConfiguration];
+
+      // Possible values for presets are hard-coded in FLT interface having
+      // corresponding AVCaptureSessionPreset counterparts.
+      // If _resolutionPreset is not supported by camera there is
+      // fallback to lower resolution presets.
+      // If none can be selected there is error condition.
+      if (![self setCaptureSessionPreset:_resolutionPreset withError:error]) {
+        [_videoCaptureSession commitConfiguration];
+        [_captureDevice unlockForConfiguration];
+        return nil;
+      }
+
+      _captureDevice.activeVideoMinFrameDuration = CMTimeMake(1, [_fps intValue]);
+      _captureDevice.activeVideoMaxFrameDuration = CMTimeMake(1, [_fps intValue]);
+
+      [_videoCaptureSession commitConfiguration];
+      [_captureDevice unlockForConfiguration];
+    } else if (outError) {
+      NSLog(@"error locking device for frame rate change (%@)", outError);
```

Comment:
Why is this being logged?

Reply:
Log removed now. Used in CI/CD test result inspections...